Dockerfile

```diff
@@ -1,15 +1,30 @@
 # Our base image
-FROM node:16-alpine
+FROM node:16-alpine as base
 
 # Create app directory
 WORKDIR /src
-RUN apk update && apk add curl tar git bash
 
+RUN apk add --no-cache git
+
+FROM base AS deps
+# Install *only* the production dependencies
 COPY ["package.json", "yarn.*", "./"]
+RUN yarn
```

Comment:
Do we need this line? or rather shouldn’t we run yarn after line 23 and delete this line?
possibly using `yarn install --production=true`

Because otherwise if you can’t install the dependancies you also can‘t get a shell and so you are stuck.